JS/index.js

```diff
@@ -1,37 +1,43 @@
+        new PassengerPlane({model: 'Sukhoi Superjet 100', maxSpeed: 870, maxFlightDistance: 11500, maxLoadCapacity: 50500, passengersCapacity: 140}),
+        new PassengerPlane({model: 'Bombardier CS300', maxSpeed: 920, maxFlightDistance: 11000, maxLoadCapacity: 60700, passengersCapacity: 196}),
+        new MilitaryPlane({model: 'B-1B Lancer', maxSpeed: 1050, maxFlightDistance: 21000, maxLoadCapacity: 80000, militaryType: MilitaryType.BOMBER}),
+        new MilitaryPlane({model: 'B-2 Spirit', maxSpeed: 1030, maxFlightDistance: 22000, maxLoadCapacity: 70000, militaryType: MilitaryType.BOMBER}),
+        new MilitaryPlane({model: 'B-52 Stratofortress', maxSpeed: 1000, maxFlightDistance: 20000, maxLoadCapacity: 80000, militaryType: MilitaryType.BOMBER}),
+        new MilitaryPlane({model: 'F-15', maxSpeed: 1500, maxFlightDistance: 12000, maxLoadCapacity: 10000, militaryType: MilitaryType.FIGHTER}),
+        new MilitaryPlane({model: 'F-22', maxSpeed: 1550, maxFlightDistance: 13000, maxLoadCapacity: 11000, militaryType: MilitaryType.FIGHTER}),
+        new MilitaryPlane({model: 'C-130 Hercules', maxSpeed: 650, maxFlightDistance: 5000, maxLoadCapacity: 110000, militaryType: MilitaryType.TRANSPORT}),
+        new ExperimentalPlane({model: "Bell X-14", maxSpeed: 277, maxFlightDistance: 482, maxLoadCapacity: 500, type: ExperimentalTypes.highAltitude, classificationLevel: ClassificationLevel.SECRET}),
+        new ExperimentalPlane({model: "Ryan X-13 Vertijet", maxSpeed: 560, maxFlightDistance: 307, maxLoadCapacity: 500, type: ExperimentalTypes.VERTICAL_TAKE_OFF_AND_LANDING, classificationLevel: ClassificationLevel.TOP_SECRET})
     ];
-
+   
     let airport = new Airport(planes);
     let militaryAirport = new Airport(airport.getMilitaryPlanes());
-    let passengerAirport = new Airport(airport.getPasPl());
-    console.log(`Military airport sorted by max distance: ${Airport.print(militaryAirport.sortByMaxDistance())}`);
-    console.log(`Passenger airport sorted by max speed: ${Airport.print(passengerAirport.sortByMaxSpeed())}`);
-    console.log(`Plane with max passenger capacity: ${Airport.print(passengerAirport.getPassengerPlaneWithMaxPassengersCapacity())}`);
-})();
\ No newline at end of file
+    let passengerAirport = new Airport(airport.getPassengerPlanes());
+    console.log(`Military airport sorted by max distance: ${printObjectToString(militaryAirport.sortByMaxDistance())}`);
+    console.log(`Passenger airport sorted by max speed: ${printObjectToString(passengerAirport.sortByMaxSpeed())}`);
+    console.log(`Plane with max passenger capacity: ${printObjectToString(passengerAirport.getPassengerPlaneWithMaxPassengersCapacity())}`);
+    console.log(`list of military planes with transport type: ${printObjectToString(militaryAirport.getTransportMilitaryPlanes())}`);
+    console.log(`List of Bomber planes In Military Planes: ${printObjectToString(militaryAirport.getBomberMilitaryPlanes())}`);
```

Comment:
Why did you add these two lines?
There were no any in the initial code version.